packages/circuit-ui/tsconfig.json

```diff
@@ -9,16 +9,14 @@
     "isolatedModules": true,
     "jsx": "react-jsx",
     "jsxImportSource": "@emotion/react",
-    "module": "ES2020",
     "moduleResolution": "node",
     "noFallthroughCasesInSwitch": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "outDir": "./dist/es",
     "resolveJsonModule": true,
-    "skipLibCheck": true,
     "strict": true,
-    "target": "es2020",
+    "target": "es2017",
```

Comment:
Downgrading the target to es2017 to match our actual browser support policy better. es2020 includes features such as optional chaining that we want to transpile.